crates/stages/src/stages/headers.rs

```diff
@@ -160,7 +174,7 @@ impl<D: HeaderDownloader, C: Consensus, H: HeadersClient, S: StatusUpdater>
         let td: U256 = *db
             .get::<tables::HeaderTD>(block_key)?
             .ok_or(DatabaseIntegrityError::TotalDifficulty { number: height })?;
-        // self.client.update_status(height, block_key.hash(), td);
+        // TODO: This should happen in the last stage
```

Comment:
This is OK here but refactoring wise let's pull it in the Finish stage when it's done